src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/tests/Router.test.js

```diff
@@ -158,30 +158,6 @@ test('Update observable attribute on route change', () => {
     expect(history.location.pathname).toBe('/list/de');
 });
 
-test('Navigate to route without default attribute when default observable is set', () => {
```

Comment:
Due to refactoring the place where the default attributes are set this test wasn't working anymore... And when thinking about it, this setup doesn't really make a lot of sense anyway. A default value for a binding like this only makes sense if the parameter is optional IMO.